src/main/java/com/exalttech/trex/ui/controllers/MainViewController.java

```diff
@@ -464,14 +465,17 @@ private TreeItem createPortItem(Port port) {
      */
     @FXML
     public void handleTreeClicked(MouseEvent mouseEvent) {
-
+        rightClickDevicesTreeMenu.ifPresent(ContextMenu::hide);
         CustomTreeItem selected = (CustomTreeItem) devicesTree.getSelectionModel().getSelectedItem();
         if (selected != null) {
-            // mouse left button clicked
+            // mouse right button clicked
```

Comment:
I think it's better to remove this comment at all. We have constant `MouseButton.SECONDARY`, it's enough, since user can easily swap secondary and primary buttons.